parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -143,107 +138,100 @@ downloadAzureCNI() {
   retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${VNET_CNI_PLUGINS_URL} || exit 41
 }
 ensureAPMZ() {
-  local version=$1
-  local apmz_url="https://upstreamartifacts.azureedge.net/apmz/$version/binaries/apmz_linux_amd64.tar.gz" apmz_filepath="/usr/local/bin/apmz"
+  local ver=$1 v
+  local d="$APMZ_DOWNLOADS_DIR/$ver"
```

Comment:
This `d` could be appended to the previous `local`.

Reply:
It can't!

You'll get an unset variable error during VHD CI

Reply:
Because it isn't serially assigning the variables, per shell statement command on that line. Or something like that.

Reply:
Ah yes, I see why.